### PR TITLE
Portfolio: Contact form

### PR DESCRIPTION
**Pull request by Devtiwo**
In this project, I :

- Made changes to the about me section.
- Worked with my coding partner in adding contact me form
- Added more images to the images folder
- Added styles for the contact me section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -117,33 +117,49 @@ <h2>About Me</h2>
...
                  </li>
              </ul>
          </article>
      </section>
+     <section class="contact">
+         <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+         <form action="https://formspree.io/f/xzbopjlq" method="POST">
+             <input type="text" name="user_fullname" placeholder="Full name" maxlength="30" required>
+             <input type="email" name="user_email" placeholder="Email address" required>
+             <textarea name="message" maxlength="500" required>Enter text here</textarea>
```
Comment (Rustamxon7) [inline index.html:152]: - [ ] You have used placeholder in other elements inside form, why don't you use also for `<textarea>`? Please kindly remove text and closing tag and enter your `Enter text here` inside placeholder like this `<textarea name="message" maxlength="500" placeholder="Enter text here" required>`. If you put your text inside placeholder your client will be able to fill the form without removing old text again 😊🙌

You can see the difference between what happens if I want to type something inside textarea

| Yours | Original |
  | ----------- | ----------- |
| image |image|

**Review by Rustamxon7 — CHANGES_REQUESTED** [review]
> ## STATUS: CHANGES REQUIRED 🟡
> Hi @joskalenda,
> 
> Great work on making the changes requested by the previous reviewer, but there are still some which aren't addressed yet.
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> really close
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShoiraTa — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Devtiwo  and @mugishasam123 ,  

You have done a great job working on required changes! Now `<textarea>` has a placeholder!
Your project is complete! Congratulations!  🎉💯🌟 

## Good Points 👍 
- Awesome UI ✔️ 
- Followed Figma template ✔️ 
- All input fields have the correct `type` attribute ✔️
- Form uses proper `action` and `method` attributes ✔️
- All input fields have correct associated placeholder text. ✔️



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -149,7 +149,7 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
          <form action="https://formspree.io/f/xzbopjlq" method="POST">
              <input type="text" name="user_fullname" placeholder="Full name" maxlength="30" required>
              <input type="email" name="user_email" placeholder="Email address" required>
-             <textarea name="message" maxlength="500" required>Enter text here</textarea>
+             <textarea name="message" maxlength="500"  placeholder="Enter text here" required></textarea>
              <button type="submit">Get in touch</button>
```
Comment (banlon-jones) [inline index.html:153]: Great job, @Devtiwo making the **Textarea** tag required this way users must type their message
## impressive

File: index.html [hunk 3]
```diff
@@ -146,11 +146,11 @@ <h2>About Me</h2>
      </section>
      <section class="contact">
          <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-         <form action="" method="POST">
+         <form action="https://formspree.io/f/xzbopjlq" method="POST">
              <input type="text" name="user_fullname" placeholder="Full name" maxlength="30" required>
              <input type="email" name="user_email" placeholder="Email address" required>
-             <textarea maxlength="500" required>Enter text here</textarea>
```
Comment (banlon-jones) [inline index.html]: great job, including field constraints like **maxlength** and **required**

**Review by banlon-jones — COMMENTED** [review]
# Excellent work @Devtiwo 

Your code is excellent and you have clearly demonstrated your understanding of the project.
However there are some changes to make, please check my comments below and make all required changes. submit for re-review when done
It was a pleasure reviewing your work, and we look forward to seeing future submissions from you.

## Project highlights:
- All linter checks are passing :heavy_check_mark:
- Your PR has a descriptive title/summary of the implemented changes :memo:
-  Descriptive commits and correct use of the GitHub Flow :fire:

[optional] Great job, adding the alt attribute in the image tag however alt` attribute should be descriptive this is because the text is displayed when the image can't load because of slow connection, and the screen reader can read the text to visually impaired users to better understand an on-page image 

## :+1: :+1: 
[OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

**Pull request merged**